src-vue/src/plugins/store.ts

```diff
@@ -172,7 +173,13 @@ export const store = createStore<FlightCoreStore>({
                 }
             }
         },
-        async launchGame(state: any, no_checks = false) {
+        async launchGame(state: any, launch_options: LaunchOptions | null = null) {
+            let no_checks = false;
+
+            if (launch_options != null) {
+                no_checks = launch_options.no_checks;
```

Comment:
Instead of manually extracting `LaunchOptions` members one by one, couldn't we pass options directly to backend using a Rust struct and its associated Typescript binding?

Reply:
Yeah, that was the longterm goal but I realise now, that I might as well do it already from the get-go ^^

Reply:
-> #798 for passing the object directly to backend as a whole ^^